### PR TITLE
[Feature-4318][API] Rename tenant code

### LINKED ISSUE #4318: [Feature][API] Rename tenant code
This'tenant code' is ambiguous and easily misleading.
As shown below:

image

'Tenant Code' is actually'Operating System User'

It is recommended to rename to:'operating system tenant'

-------

这个'租户编码' 存在歧义啊, 容易产生误导.
如下图:

image

'租户编码' 实际上是 '操作系统用户'

建议重命名为 :  '操作系统租户' .

### PR DESCRIPTION
**Pull request by zhanguohao**
## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/index.html before opening a pull request.*

## What is the purpose of the pull request
 
   Rename tenant code
  This closes #4318 

## Verify this pull request

This change added tests and can be verified as follows:

  - *Manually verified the change by testing locally.*

### CODE REVIEW

**Comment by CalvinKirs** [comment]
It would be nice to change tenant_code to OS_TENANT as well.PTAL? @BoYiZhang

**Comment by BoYiZhang** [comment]
> It would be nice to change tenant_code to OS_TENANT as well.PTAL? @BoYiZhang

+1

**Comment by chengshiwen** [comment]
+1

File: dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js [hunk 1]
```diff
@@ -185,14 +185,14 @@ export default {
   memoryUsage: 'memoryUsage',
   'Last heartbeat time': '最后心跳时间',
   'Edit Tenant': '编辑租户',
-  'Tenant Code': '租户编码',
```
Comment (zhuangchong) [inline dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js:188]: add  English 'OS Tenant Code ' to show?

**Comment by JinyLeeChina** [comment]
> It would be nice to change tenant_code to OS_TENANT as well.PTAL? @BoYiZhang

+1

File: dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js (hunk 1 above)
Comment (zhanguohao) [reply dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js:188]: I‘ll adjust it right away

**Comment by break60** [comment]
It’s best to correspond to the names before and after, and then the page is also changed
image

**Comment by zhanguohao** [comment]
> It’s best to correspond to the names before and after, and then the page is also changed

Get.

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 2]
```diff
@@ -36,7 +36,7 @@
     USER_NAME_NULL(10004, "user name is null", "用户名不能为空"),
     HDFS_OPERATION_ERROR(10006, "hdfs operation error", "hdfs操作错误"),
     TASK_INSTANCE_NOT_FOUND(10008, "task instance not found", "任务实例不存在"),
-    TENANT_CODE_EXIST(10009, "tenant code {0} already exists", "租户编码[{0}]已存在"),
+    TENANT_CODE_EXIST(10009, "os tenant code {0} already exists", "操作系统租户[{0}]已存在"),
```
Comment (CalvinKirs) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java:39]: It is best to modify all TENANT_CODE_EXIST to OS_TENANT_CODE_EXIST

Comment (zhanguohao) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java:39]: Got it.

**Review by break60 — APPROVED** [review]
+1

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 3]
```diff
@@ -116,7 +116,7 @@
     QUERY_TENANT_LIST_ERROR(10086, "query tenant list error", "查询租户列表错误"),
     UPDATE_TENANT_ERROR(10087, "update tenant error", "更新租户错误"),
     DELETE_TENANT_BY_ID_ERROR(10088, "delete tenant by id error", "删除租户错误"),
-    VERIFY_TENANT_CODE_ERROR(10089, "verify tenant code error", "租户编码验证错误"),
+    VERIFY_TENANT_CODE_ERROR(10089, "verify os tenant code error", "操作系统租户验证错误"),
```
Comment (CalvinKirs) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java:119]: hi,suggest change

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 4]
```diff
@@ -151,7 +151,7 @@
     ENCAPSULATION_PROCESS_INSTANCE_GANTT_STRUCTURE_ERROR(10121, "encapsulation process instance gantt structure error", "查询工作流实例甘特图数据错误"),
     QUERY_PROCESS_DEFINITION_LIST_PAGING_ERROR(10122, "query process definition list paging error", "分页查询工作流定义列表错误"),
     SIGN_OUT_ERROR(10123, "sign out error", "退出错误"),
-    TENANT_CODE_HAS_ALREADY_EXISTS(10124, "tenant code has already exists", "租户编码已存在"),
+    TENANT_CODE_HAS_ALREADY_EXISTS(10124, "os tenant code has already exists", "操作系统租户已存在"),
```
Comment (CalvinKirs) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java:154]: hi,suggest change

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 5]
```diff
@@ -195,7 +195,7 @@
     QUERY_WORKFLOW_LINEAGE_ERROR(10161, "query workflow lineage error", "查询血缘失败"),
     QUERY_AUTHORIZED_AND_USER_CREATED_PROJECT_ERROR(10162, "query authorized and user created project error error", "查询授权的和用户创建的项目错误"),
     DELETE_PROCESS_DEFINITION_BY_ID_FAIL(10163,"delete process definition by id fail, for there are {0} process instances in executing using it", "删除工作流定义失败，有[{0}]个运行中的工作流实例正在使用"),
-    CHECK_TENANT_CODE_ERROR(10164, "Please enter the English tenant code", "请输入英文租户编码"),
+    CHECK_TENANT_CODE_ERROR(10164, "Please enter the English os tenant code", "请输入英文操作系统租户"),
```
Comment (CalvinKirs) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java:198]: hi,suggest change

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java (hunk 3 above)
Comment (CalvinKirs) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java:119]: TENANT_CODE is recommended to change to OS_TENANT_CODE

Comment (zhanguohao) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java:119]: got it

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java (hunk 4 above)
Comment (zhanguohao) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java:154]: got it

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java (hunk 5 above)
Comment (zhanguohao) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java:198]: got it

**Review by CalvinKirs — APPROVED** [review]
+1

**Comment by CalvinKirs** [comment]
considering that it's your first contribution, I think we can get deep communiction, you can contact me through mail or add wechat(Kris_Evil), when mail or added, please tell me who you are, I think I can help to familiar with the DolphinScheduler if you meet with problems.

**Pull request merged**